indigo/distribution.yaml

```diff
@@ -8732,6 +8732,11 @@ repositories:
       url: https://github.com/WPI-RAIL/ros_ethernet_rmp.git
       version: develop
     status: maintained
+  ros_in_hand_scanner:
+    doc:
+      type: git
+      url: https://github.com/RodBelaFarin/ros_in_hand_scanner.git
+      version: developed
```

Comment:
Yes, you need to update this PR to contain the intended content.

The easiest way to do that is by committing the fix to the branch this PR has been created from.